### DISCUSSION TITLE
No Update - message FAILED The form does not contain the default language (it-IT).

### DISCUSSION THREAD

**Discussion by sales57** [YetiForceCompany/YetiForceCRM#15769] [How to]
Hi

I have proble with update yetiforce  6.1  to 6.2 
Message is " FAILED The form does not contain the default language (it-IT).

What i must do ?

Please give me information 

Thank you

Roberto

**Comment by mariuszkrzaczkowski** [comment]
What package are you installing and where did you download it from?

**Reply by sales57** [reply]
I have package 6.1.0  installed by softcoulus  
Call upgrade by package Yetiforce 

Thank you for you time dedicate me

**Reply by mariuszkrzaczkowski** [reply]
Try this package
https://github.com/YetiForceCompany/UpdatePackages/tree/master/YetiForce%20CRM%206.x.x/6.1.0_to_6.2.0/zip

**Reply by sales57** [reply]
all 3 files o only .zip ?

**Reply by mariuszkrzaczkowski** [reply]
only .zip

**Reply by sales57** [reply]
No update from program 
where i must put .zip with ftp  which directory.
and after this operation ?  what i must do ?

**Reply by mariuszkrzaczkowski** [reply]
Read the documentation, everything is described here
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/system-update

**Comment by mariuszkrzaczkowski** [comment]
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/system-update

**Reply by sales57** [reply]
I have download package .zip
From program does not load .zip
I can update with ftp
where to place the file to make it visible to the program?
